### ISSUE TITLE
Problem with Rhvoice.su website

### ISSUE THREAD

**Issue by Giovani93** [RHVoice/marianna-ukr#1]
Hello.
My name is Marco Oros. I'd like to download new voices from Rhvoice.su website. Actually, this website doesn't work. I'd like to download those voices for My screen reader NVDA.
Thank You.
Best regards
Marco Oros

**Comment by zstanecic** [comment]
Hi,

Rhvoice.su has technical issues. Please be patient, 

 

From: Giovani93 ***@***.***> 
Sent: Sunday, September 24, 2023 4:24 PM
To: RHVoice/marianna-ukr ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [RHVoice/marianna-ukr] Problem with Rhvoice.su website (Issue #1)

 

Hello.
My name is Marco Oros. I'd like to download new voices from Rhvoice.su website. Actually, this website doesn't work. I'd like to download those voices for My screen reader NVDA.
Thank You.
Best regards
Marco Oros
